lib/swagger/paths.js

```diff
@@ -38,85 +38,113 @@ swaggerPaths._addPathDefinition = function(paths, resourceConfig) {
       description: "Search for " + resourceName,
       parameters: resourceConfig.searchParams,
       hasPathId: false
-    }),
-    post: swaggerPaths._getPathOperationObject({
+    });
+  }
+
+  if (resourceConfig.handlers.create) {
+    genericPaths.post = swaggerPaths._getPathOperationObject({
       handler: "create",
       resourceName: resourceName,
       description: "Create a new instance of " + resourceName,
       parameters: resourceConfig.attributes,
       hasPathId: false
-    })
-  };
+    });
+  }
 
-  paths["/" + resourceName + "/{id}"] = {
-    get: swaggerPaths._getPathOperationObject({
+  if (resourceConfig.handlers.find) {
+    specificPaths.get = swaggerPaths._getPathOperationObject({
       handler: "find",
       resourceName: resourceName,
       description: "Get a specific instance of " + resourceName,
       hasPathId: true
-    }),
-    delete: swaggerPaths._getPathOperationObject({
-      handler: "delete",
-      resourceName: resourceName,
-      description: "Delete an instance of " + resourceName,
-      hasPathId: true
-    }),
-    patch: swaggerPaths._getPathOperationObject({
+    });
+  }
+
+  if (resourceConfig.handlers.delete) {
+    specificPaths.delete = swaggerPaths._getPathOperationObject({
+        handler: "delete",
+        resourceName: resourceName,
+        description: "Delete an instance of " + resourceName,
+        hasPathId: true
+    });
+  }
+
+  if (resourceConfig.handlers.update) {
+    specificPaths.patch = swaggerPaths._getPathOperationObject({
       handler: "update",
       resourceName: resourceName,
       description: "Update an instance of " + resourceName,
       hasPathId: true
-    })
-  };
+    });
+  }
 };
 
 swaggerPaths._addDeepPaths = function(paths, resourceName, resourceConfig, relationName, relation) {
-  paths["/" + resourceName + "/{id}/" + relationName] = {
-    get: swaggerPaths._getPathOperationObject({
-      handler: "find",
-      resourceName: relation,
-      hasPathId: true
-    })
-  };
+  if (resourceConfig.handlers.find) {
+    paths["/" + resourceName + "/{id}/" + relationName] = {
+      get: swaggerPaths._getPathOperationObject({
+        handler: "find",
+        resourceName: relation,
+        hasPathId: true
+      })
+    };
+  }
 
   var relationType = resourceConfig.attributes[relationName]._settings.__many ? "many" : "one";
-  paths["/" + resourceName + "/{id}/relationships/" + relationName] = {
-    get: swaggerPaths._getPathOperationObject({
+  var relationPaths = { }
+  paths["/" + resourceName + "/{id}/relationships/" + relationName] = relationPaths;
+
+  if (resourceConfig.handlers.find) {
+    relationPaths.get = swaggerPaths._getPathOperationObject({
       handler: "find",
       resourceName: relation,
       relationType: relationType,
       extraTags: resourceName,
       hasPathId: true
-    }),
-    post: swaggerPaths._getPathOperationObject({
+    });
+  }
+
+  if (resourceConfig.handlers.update) {
```

Comment:
This is meant to be an `update` - the path handles the creation of a relationship on a resource, which is actually an `update`